packages/lesswrong/server/migrations/20251119T184455.dropUserLoginTokensMaterializedView.ts

```diff
@@ -0,0 +1,9 @@
+
+export const up = async ({db}: MigrationContext) => {
+  await db.none(`
+    DROP MATERIALIZED VIEW fm_get_user_by_login_token
```

Comment:
We probably also want to drop the login token indexes deleted from [usersDbIndexes.ts‎](https://github.com/ForumMagnum/ForumMagnum/pull/11722/files#diff-2915d265587f712d0d81777eceb2ce69e1e4079344047a0be0c22ebc7e8a03f1)